### ISSUE TITLE
output_stride parameter 

### ISSUE THREAD

**Issue by Darcy103** [justld/contrast_seg_paddle#1]
I have a confusion to consult you, can you explain this parameter **_output_stride_**? I don't understand how to ensure that the H and W dimensions of the two parameters of **_feats__****_(embedding)_** and **_predict(seg)_** are the original input size when the backbone network is ResNet.

**Comment by justld** [comment]
Hello, hrnet_contrast only support backbone HRNet_W48 in PaddleSeg.

In pixle_contrast_cross_entropy_loss.py，labels are downsampled to the dimensions of 'embedding' and 'seg'.

In pixle_contrast_cross_entropy_loss.py


        def contrast_criterion(self, feats, labels=None, predict=None):
                labels = labels.unsqueeze(1)
                labels = F.interpolate(labels, feats.shape[2:], mode='nearest')  <<<====look here
                labels = labels.squeeze(1)

                batch_size = feats.shape[0]
                labels = labels.reshape((batch_size, -1))
                predict = predict.reshape((batch_size, -1))
                feats = paddle.transpose(feats, [0, 2, 3, 1])
                feats = feats.reshape((feats.shape[0], -1, feats.shape[-1]))

                feats_, labels_ = self._hard_anchor_sampling(feats, labels, predict)

                loss = self._contrastive(feats_, labels_)
                return loss

**Comment by Darcy103** [comment]
I understand, thanks for your explanation.
